experimental/canvas/explorations-py/.pylintrc

```diff
@@ -0,0 +1,11 @@
+[MESSAGES CONTROL]
+disable = 
+    C0330,
+    C0326,
+    fixme,
+    line-too-long,
+    missing-function-docstring,
+    missing-module-docstring
```

Comment:
We should start putting docstrings onto our modules, classes, and functions. Don't have to fix them all in this commit though. Can be done in the next canvas-related PR.